src/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+add_subdirectory(Shaders)
+add_subdirectory(VmaReplay)
```

Comment:
This always builds `VmaReplay`. As a consumer of VMA, I don't want to build the replay application with my application. Please give me a CMake option to disable building both this and the shaders

Reply:
Done.